CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 ##### Fixes :wrench:
 
+- Expanded the integration with the [iTwin Platform](https://developer.bentley.com/) to load GeoJSON and KML data from the Reality Management API. Use `ITwinData.createDataSourceForRealityDataId(iTwinId, realityDataId)` to load data as the corresponding Data Source. [#12344](https://github.com/CesiumGS/cesium/pull/12344)
```

Comment:
Does this belong under `Additions` rather than `Fixes`?

Reply:
:facepalm: yes, fixed